src/krati_Jain/BankingSystem.java

```diff
@@ -0,0 +1,97 @@
+/*Assignment 9* : *2nd April 2021*
+
+*Create a Banking System which has the following functionality.*
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+*Output* :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+*Output* :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+
+package krati_Jain;
+
+
+public class BankingSystem {
+	
+	String customerName;
+	double accountBalance;
+	static int overallCredits;
+	static int overallDebits;
+	static int totalPrintBalance;
+	int printBalance;
+	int totalDebits;
+	int totalCredits;
+	
+	void customerDetails(String custName, double actBalanace) {
+		customerName = custName;
+		accountBalance = actBalanace;
+	}
+	
+	void debitAmount(double debitAmt) {
+		while(accountBalance >= debitAmt) {
```

Comment:
Why while loop here?
you should write if condition to check Whether balance is sufficient or not.